index.html

```diff
@@ -343,9 +348,11 @@ <h2>Uber Navigation</h2>
             </ul>
             <button class="project-button">See Project</button>
```

Comment:
- [ ] While trying to test for the button contrast check of your page, I noticed that the `See Project` button is failing for the hover effect. Kindly check it and make it pass. You can use this [button contrast checker](https://www.aditus.io/button-contrast-checker/) to test for the button contrast of your page

![hover-failure](https://user-images.githubusercontent.com/26597432/193934097-914d1039-8f4d-405e-8923-96d59a9788b1.png)